src/common/options.rs

```diff
@@ -412,29 +586,52 @@ pub struct Keyword {
     pub intensifier: Option<f64>,
 }
 
+/// Used as a parameter for [`OptionsBuilder::utterances`].
+///
+/// See the [Deepgram Utterances feature docs][docs] for more info.
+///
+/// [docs]: https://developers.deepgram.com/docs/utterances
 #[derive(Debug, PartialEq, Clone, Copy)]
-enum Utterances {
+#[non_exhaustive]
+pub enum Utterances {
+    #[allow(missing_docs)]
     Disabled,
-    Enabled { utt_split: Option<f64> },
```

Comment:
Here's an example of us using `Enabled` and `Disabled` instead of `Enabled(bool)`.  Granted, this one also attaches other data to the `Enabled` option, so it's not a perfect analogy.